scripts/Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM alpine
 MAINTAINER chriseth <chris@ethereum.org>
+#Official solidity docker image
 
-RUN \
-  apk --no-cache --update add build-base cmake boost-dev git                                                && \
-  sed -i -E -e 's/include <sys\/poll.h>/include <poll.h>/' /usr/include/boost/asio/detail/socket_types.hpp  && \
-  git clone --depth 1 --recursive -b develop https://github.com/ethereum/solidity                           && \
-  cd /solidity && cmake -DCMAKE_BUILD_TYPE=Release -DTESTS=0 -DSTATIC_LINKING=1                             && \
-  cd /solidity && make solc && install -s  solc/solc /usr/bin                                               && \
-  cd / && rm -rf solidity                                                                                   && \
-  apk del sed build-base git make cmake gcc g++ musl-dev curl-dev boost-dev                                 && \
-  rm -rf /var/cache/apk/*
+#Establish working directory as solidity
+WORKDIR /solidity
+#Copy working directory on travis to the image
+COPY / $WORKDIR
```

Comment:
The dockerfile itself is not important to the user. What is important is the image that it produces. This enables us to easily automate the creation of an image.